common/v2/features/AddAccount/components/DeterministicWallets.tsx

```diff
@@ -19,6 +19,9 @@ import nextIcon from 'assets/images/next-page-button.svg';
 import prevIcon from 'assets/images/previous-page-button.svg';
 import radio from 'assets/images/radio.svg';
 import radioChecked from 'assets/images/radio-checked.svg';
+import { getNetworkByDPath } from 'v2/services/Store/Network/helpers';
+import { Network } from 'v2/types';
+import { StaticNetworkConfig, CustomNetworkConfig } from 'shared/types/network';
```

Comment:
**convention:** Only import from `v2`
FYI: Please only import from the `v2` directory. 
- if something is missing you are welcome to add it into the v2 dir
- in this case v2 type is actually `Network`